Sources/Views/ContentTile/ServicesTileViewModel.swift

```diff
@@ -0,0 +1,36 @@
+import GDSAnalytics
+import GDSCommon
+import Logging
+import UIKit
+
+struct ServicesTileViewModel: GDSContentTileViewModel,
+                              GDSContentTileViewModelWithBody,
+                              GDSContentTileViewModelWithSecondaryButton {
+    let title: GDSLocalisedString = "app_yourServicesCardTitle"
+    let body: GDSLocalisedString = "app_yourServicesCardBody"
+    let showSeparatorLine: Bool = true
+    let secondaryButtonViewModel: ButtonViewModel
+    let backgroundColour: UIColor? = .secondarySystemGroupedBackground
+    
+    init(analyticsService: AnalyticsService,
+         action: @escaping () -> Void) {
+        let event = LinkEvent(textKey: "app_yourServicesCardLink",
+                              linkDomain: AppEnvironment.yourServicesLink,
+                              external: .false)
```

Comment:
I think this is an external event, it's external to the Mobile Pod at least. Or are we defining external as outside of One Login?

Reply:
Looks like it's for outside GDS or potentially outside of government? Either way, I think we can be sure this one isn't 🙂 
<img width="649" alt="image 2024-10-07 at 23 36 49" src="https://github.com/user-attachments/assets/cf95806f-18b3-45be-9a0d-cabbaa2c010c">

Reply:
works for me - thanks!